xml/release-notes.xml

```diff
@@ -454,6 +454,22 @@
     </para>
    </sect2>
 
+   <sect2 xml:id="ibus-layout-name-change">
+    <title>Manual config migration of IBus is necessary due to layout name change</title>
+    <para>
+     Since IBus 1.5.23 renames some keyboard layouts, it cannot load configuration
+     containing these renamed layouts after upgrading. Thereby, it might change the
+     layout to US. The renamed layouts include be:*:ger, ch:*:ger, de:*:ger,
+     gr:*:gre, ro:*:rum, and, sk:*:slo.
+     See <link xlink:href="https://bugzilla.opensuse.org/show_bug.cgi?id=1177545"/>.
+     </para>
+     <para>
+     Users need to migrate configuration manually. Please open GNOME Settings and
```

Comment:
We try to avoid using words like `please` in the documentation.
```suggestion
     Users need to migrate configuration manually. Open GNOME Settings and
```

Reply:
I will fix it.